### ISSUE TITLE
Screenreader + aria-owns issue

### ISSUE THREAD

**Issue by CNDexter** [FrDH/mmenu-js#1193]
You are unable to use a screenreader if you have the "close" element enabled on the navbar addon. It seems that aria-owns for the button is pointing to the parent item, when it should in fact point to the children items in the menu. 

You can see this not working on demo/advanced.html In my project I override the js locally so that it points to the children items, like above, I am able to use the screenreader on the site. 

Working example: 
<img width="633" alt="aria-own" src="https://user-images.githubusercontent.com/3587944/98389821-92df0b80-2022-11eb-9915-fbc577eda6b7.png">

**Comment by FrDH** [comment]
Thanks, ill have a look at this.

**Comment by CNDexter** [comment]
@FrDH The close menu is working, but you can't read through the page with a screen reader. If you remove aria-owns it will work, or if you update it to select the children menu items it will work so it seems that the problem is the aria-owns association.

I can create a video demonstrating the issue later today.

**Comment by FrDH** [comment]
Ok I understand now. I think the close button can go without the aria-owns attribute. Could you check that?
